lib/bundler/rubygems_ext.rb

```diff
@@ -29,6 +29,29 @@
 end
 
 module Gem
+
+  BUNDLED_GEMS = {
```

Comment:
I don't think this should be part of Bundler. Bundle is not the one who decides that some gem is default/bundled. It is Ruby upstream decision.